scripts/test.sh

```diff
@@ -45,6 +45,10 @@ golang () {
   echo "Testing Golang..."
 
   dir="clients/${PROJECT}/go"
+  if ! [ -f README.md ]; then
+    echo "package main" > "$dir/main.go"
+    echo "func main(){}" >> "$dir/main.go"
```

Comment:
How about

```
(cd "${dir}/client" && go mod tidy && go build -o "$(mktemp)" .)
(cd "${dir}/models" && go mod tidy && go build -o "$(mktemp)" .)
```

instead?

Reply:
I don't understand the question?

Your patch doesn't test the build, it just tests an empty function. My suggestion builds client and models directories so we test the build actually (if the code is buildable)

Reply:
Will there only ever be those two directories, or could we have more that are then missed by that test?

Reply:
only directories for swagger generated specs (not openapi3)